src/algebra/lie_algebra.lean

```diff
@@ -205,16 +214,209 @@ def of_associative_algebra (A : Type v) [ring A] [algebra R A] : lie_algebra R A
 An important class of Lie algebras are those arising from the associative algebra structure on
 module endomorphisms.
 -/
-def of_endomorphism_algebra (M : Type v)
+instance of_endomorphism_algebra (M : Type v)
   [add_comm_group M] [module R M] : lie_algebra R (module.End R M) :=
-of_associative_algebra R (module.End R M)
+of_associative_algebra (module.End R M)
+
+@[simp] lemma endo_algebra_bracket (M : Type v)
```

Comment:
```suggestion
lemma endo_algebra_bracket (M : Type v)
```
I'm not sure if this should always be `simp`ed away. I think it's better to let the user add it to the simp set manually. Because once you've simped away the Lie bracket, other simp lemmas involving the bracket will no longer fire.

Reply:
Very good point, agreed.